redash/utils/__init__.py

```diff
@@ -74,7 +74,9 @@ class JSONEncoder(json.JSONEncoder):
     """Adapter for `json.dumps`."""
 
     def __init__(self, **kwargs):
-        self.encoders = [m.custom_json_encoder for m in sys.modules.values() if hasattr(m, "custom_json_encoder")]
+        from redash.query_runner import query_runners
+
+        self.encoders = [r.custom_json_encoder for r in query_runners.values() if hasattr(r, "custom_json_encoder")]
```

Comment:
As you can see, this change is much smaller and more readable than previously: only the essential was done to fix the issue at hand. We can move things away from the `__init__.py` files in a later PR, but for now we need to fix the CI, so we shouldn't do any refactor changes if they're not necessary.